integrated_channels/sap_success_factors/migrations/0001_squashed_0022_auto_20200206_1046.py

```diff
@@ -22,7 +22,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('sapsf_user_id', models.CharField(max_length=255)),
-                ('enterprise_course_enrollment_id', models.PositiveIntegerField(db_index=True)),
+                ('enterprise_course_enrollment_id', models.IntegerField(db_index=True)),
```

Comment:
Looks good. 
1. Can you confirm if this passes your mysql8 test?
2. Maybe add comments to the squash migration files (or wherever you think) stating that the replaced migrations should not be deleted until after nutmeg. I guess it could go in the replaced migration files as well.

Reply:
1: Yes now migration checks work fine with mysql8. Please check these migrations here with mysql8 [blackboard](https://github.com/openedx/edx-platform/runs/4965788270?check_suite_focus=true#step:11:150) and [sap](https://github.com/openedx/edx-platform/runs/4965788270?check_suite_focus=true#step:11:614).

2. Adding comments